src/native/fs/fs_napi.cpp

```diff
@@ -45,6 +46,7 @@ const static std::map<std::string, int> flags_to_case = {
     { "rs+", O_RDWR | O_SYNC },
     { "sr+", O_RDWR | O_SYNC },
     { "w", O_TRUNC | O_CREAT | O_WRONLY },
+    { "wt", O_RDWR | O_TMPFILE },
```

Comment:
silly question, why do we need O_RDWR for tmpfile? isn't O_WRONLY enough?

Reply:
it is opened as temp, but it becomes a permanent file, you might want to read it ?

Reply:
I see, and I just wonder - is there any benefit to use write-only when possible? or there shouldn't be much different to the filesystem? (any filesystem)

Reply:
I don't think there is any benefit, and we might need read to compress, encrypt ...?

Reply:
ok got you. thanks for the explanation. resolving.